driver/esp32/espidf.c

```diff
@@ -171,12 +171,16 @@ typedef struct {
 
 #define DISPLAY_TYPE_ILI9341 1
 #define DISPLAY_TYPE_ILI9488 2
+#define DISPLAY_TYPE_GC9A01  3
+#define DISPLAY_TYPE_ST7789  4
 
 void ili9xxx_flush(void *_disp_drv, const void *_area, void *_color_p)
```

Comment:
Something is wrong with the alignment in `ili9xxx_flush`.   
I think that in your changes you added tabs instead of spaces, and it looks weird when tab is configured to 4 spaces.

Reply:
I will look at that; it should be spaces.